src/ray/protobuf/gcs.proto

```diff
@@ -408,21 +408,19 @@ message ResourcesData {
   ResourceLoad resource_load_by_shape = 7;
   // Whether this node manager is requesting global GC.
   bool should_global_gc = 8;
-  // IP address of the node.
-  string node_manager_address = 9;
```

Comment:
should we keep the same index so it's backward compatible?

Reply:
We should. But I feel right now, we don't support usage of this, it should be fine now. Definitely later if we support rolling upgrade kind of features, we should keep the same index.

Reply:
We should. But I feel right now, we don't support usage of this, it should be fine now. Definitely later if we support rolling upgrade kind of features, we should keep the same index.